arch/arm/src/cxd56xx/cxd56_cpustart.c

```diff
@@ -122,7 +122,7 @@ static void appdsp_boot(void)
   irq_attach(CXD56_IRQ_SMP_CALL, cxd56_smp_call_handler, NULL);
   up_enable_irq(CXD56_IRQ_SMP_CALL);
 
-  spin_unlock(&g_appdsp_boot);
+  raw_spin_unlock(&g_appdsp_boot);
```

Comment:
The semantics of **raw_** will be removed. Please hold this PR and do not merge.

Reply:
Should we wait for the same changes on https://github.com/espressif/esp-hal-3rdparty/pull/8?

Reply:
> Should we wait for the same changes on [espressif/esp-hal-3rdparty#8](https://github.com/espressif/esp-hal-3rdparty/pull/8)?

we don't need https://github.com/espressif/esp-hal-3rdparty/pull/8 anymore, the real fix is this line:
https://github.com/apache/nuttx/pull/15691/files#diff-322a814575a3e73db5f2836c677e324a3a9e814c42aa87ca583a4199dfa21341R103